### PR TITLE
feat(augur): augur adapters and changes

### PR DESCRIPTION
**Pull request by boxhock**

### CODE REVIEW

File: packages/composites/augur/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "**/*.spec.ts", "**/*.test.ts"],
+  "references": [{ "path": "../../core/test-helpers" }, { "path": "../../core/bootstrap" }]
```
Comment (justinkaseman) [inline packages/composites/augur/tsconfig.json:9]:
```suggestion
  "references": [
  { "path": "../../core/test-helpers" },
  { "path": "../../core/bootstrap" },
  { "path": "../../sources/therundown" }
  ]
```
Should fix your docker build problem.

Comment (boxhock) [reply packages/composites/augur/tsconfig.json]: That did it, thanks! I'll keep this in mind for next time

File: packages/sources/therundown/src/endpoint/events.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      'x-rapidapi-key': config.apiKey,
+    },
+    params: {
+      include: 'scores',
+    },
+    url,
+  }
+
+  const response = await Requester.request(reqConfig)
+  if (status !== undefined) {
```
Comment (justinkaseman) [inline packages/sources/therundown/src/endpoint/events.ts:43]: In the case that `status` _is_ undefined should we be throwing an error?

If so we could pass a callback to `Requester.request` with a custom validation check for this.

File: packages/composites/augur/test/unit/pack.test.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import * as create from '../../src/methods/createMarkets'
+import * as resolve from '../../src/methods/resolveMarkets'
+
+describe('packCreation', () => {
```
Comment (justinkaseman) [inline packages/composites/augur/test/unit/pack.test.ts:4]: These look like E2E test, not unit tests. Could we rename the folder to `e2e`?

**Review by justinkaseman — APPROVED** [review]
Very clean use of our new patterns, TypeScript, and docs. LGTM.

**Review by justinkaseman — APPROVED** [review]

**Pull request merged**